internal/config/plugin_installer.go

```diff
@@ -665,7 +670,7 @@ func PluginCommand(out io.Writer, cmd string, args []string) {
 		for _, plugin := range args {
 			// check if the plugin exists.
 			for _, p := range Plugins {
-				if p.Name == plugin && p.Default {
+				if (p.Name == plugin && p.Builtin) || p.Name == "initlua" {
```

Comment:
1. Now, when _successfully_ removing _any_ plugin, micro prints "Default plugins cannot be removed..."

Although the plugin still ends up being successfully removed, thanks to `continue` (BTW I'm not sure why we have `continue` instead of `break` here).

2. What about `UpdatePlugins()`?

Reply:
Oh god. I was really hoping I wouldn't have to point out the obvious. It prints "Default plugins cannot be removed..." because you for some reason put ` || p.Name == "initlua"` outside the parentheses.

(And before you rush to move the parentheses, another obvious thing: it is completely pointless to check `p.Name == "initlua"` at each iteration of the loop. We can just check `plugin == "initlua"` before entering the loop.)

So, now that you changed `continue` to `break` (which I mentioned just as a side note), without fixing the issue that you've introduced, as a result it not just needlessly says "Default plugins cannot be removed" but also fails to remove _any_ plugins (and honestly says "No plugins removed"). Which you should have noticed if you tested your changes.